### ISSUE TITLE
multiple clauses directive with one instance for element

### ISSUE THREAD

**Issue by xrado** [vuejs/vue#274]
Wouldn't be nice to have variation of directive with one instance for element. Bind / unbind would be called only once and update on each clause.

Here is what Im trying to do. I would like to create validator directive like this 

```
v-valid="required,maxLength:10"
```

Bind would only attach event listeners (keyup,change,..) once and then update would set all the validators need to run on change. Unbind would remove event listeners.

Currently each directive clause become own instance, so validator clauses become separated, but what is needed is result of all together for the element.

Any idea how to solve this?
